core/src/main/kotlin/plugability/DokkaContext.kt

```diff
@@ -124,7 +124,7 @@ private class DokkaContextConfigurationImpl(
     }
 
     private fun findNotOverridden(bucket: Set<Extension<*, *, *>>): Extension<*, *, *> {
-        val filtered = bucket.filter { it !in suppressedExtensions }
+        val filtered = bucket.filter { it !in suppressedExtensions.filter { it.value.all { it is Suppression.ByExtension } } }
```

Comment:
I got it, thank you for the explanation.
It would be great to leave a code comment here.

```suggestion
        val suppressedExtensionsByOverrides = suppressedExtensions.filter { it.value.any { it is Suppression.ByExtension }
        val filtered = bucket.filterNot { it in suppressedExtensionsByOverrides } }
```

Reply:
Fixed